pkg/api/node/formatter.go

```diff
@@ -102,15 +113,31 @@ func (h ActionHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 	rw.WriteHeader(http.StatusNoContent)
 }
 
-func (h ActionHandler) do(rw http.ResponseWriter, req *http.Request) error {
-	vars := util.EncodeVars(mux.Vars(req))
-	action := vars["action"]
-	name := vars["name"]
+func (h ActionHandler) IsMatchedResource(resource subresource.Resource, method string) bool {
+	if resource.Name != subresource.Node.Resource || method != http.MethodPost {
```

Comment:
Again, shouldn't this be a HTTP method `PUT` for idempotent actions (such as cordon or uncordon)?
Additionally, list-actions like `listUnhealthyVM` should be a `GET`, right?

Reply:
Same reason in https://github.com/harvester/harvester/pull/5704#discussion_r1622033858.

Sure, list-actions should be `GET`.